src/kube-runtime/src/init.d/initializer.py

```diff
@@ -8,46 +8,45 @@
 # documentation files (the "Software"), to deal in the Software without restriction, including without limitation
 # the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
 # to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
 #
 # THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
 # BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
 # NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-from __future__ import print_function
-
 import os
 import sys
-import collections
 import logging
 import argparse
-import json
-import yaml
 import subprocess
 import re
 
-logger = logging.getLogger(__name__)
+import yaml
+
+LOGGER = logging.getLogger(__name__)
 
-EXIT_PLUGIN_INVALIDATE=100
+EXIT_PLUGIN_INVALIDATE = 100
 
 def run_script(script_path, parameters, plugin_scripts):
     args = [sys.executable, script_path, "{}".format(parameters)]
     args += plugin_scripts
     proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
     while True:
-        line = proc.stdout.readline()
+        line: bytes = proc.stdout.readline()
         if not line:
             break
-        line = line.decode("utf-8").strip()
-        logger.info(line)
+        line = line.decode("UTF-8").strip()
+        LOGGER.info(line)
     proc.wait()
     if proc.returncode:
-        logger.error("failed to run %s, error code is %s", script_path, proc.returncode)
+        LOGGER.error("failed to run %s, error code is %s", script_path, proc.returncode)
+        raise Exception("Failed to run init script")
 
 # just a workaround, need to remove this function in the future
-def is_plugin_validate(jobConfig):
+def is_plugin_validate(jobconfig):
```

Comment:
I know, I think it's a bug. Before this change, I can not find the ref to `jobConfig`, this args is not used. And `jobconfig` is a global variable, so it can be reference here, Nothing wrong happens. 

I remove the global variable and change to `jobconfig` here

Reply:
you can change all names in this function

Reply:
Currently, I have removed the global variable of jobconfig. and no such lint error for this file 

---
In reply to: [349433452](https://github.com/microsoft/pai/pull/3875#discussion_r349433452) [](ancestors = 349433452)

Reply:
there're two words, which should be [separated by underscores to improve readability](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)
you didn't get lint error because lint does not know there're two words

Reply:
ok, fine

---
In reply to: [349480043](https://github.com/microsoft/pai/pull/3875#discussion_r349480043) [](ancestors = 349480043)